### DISCUSSION TITLE
adding a nwdaf function

### DISCUSSION THREAD

**Discussion by henokbfg** [srsran/srsRAN_Project#766] [General Help]
Hello,

I'm new to all this and I'm looking for some help. I managed to build, run and test the srsRAN gnb with srsue tutorial. Now I want to add a basic nwdaf function to the 5g core that for now just collects metrics from the Network function such as AMF. I have no idea where to even start.

**Comment by herlesupreeth** [comment]
NWDAF is a Core Network component and this is a gNB (RAN) repository so you may want to post this question in a 5GC repository like open5gs etc
